manage_app_pool added ability to set idle timeout

lcoulson opened the pull request:
manage_app_pool added ability to set idle timeout

new params set default undef for backward compatibility
logic in class handles this

Caller set to 0 will disable this feature in IIS for given app pool

igalic commented:
same as: https://github.com/voxpupuli/puppet-iis/pull/72#issuecomment-171346231 ;)

lcoulson commented:
@igalic @juniorsysadmin now hopefully compliant, added tests, renamed variables, dealt with Robocop absurdity for splitting int with underscores every 3 digits :) ...

juniorsysadmin commented:
Looks okay to me
